src/components/Widget.tsx

```diff
@@ -114,10 +114,27 @@ export default function Widget(props: PropsWithChildren<WidgetProps>) {
     }
     return props.locale ?? DEFAULT_LOCALE
   }, [props.locale])
-  const jsonRpcEndpoint = useMemo(
-    () => props.jsonRpcEndpoint ?? JSON_RPC_FALLBACK_ENDPOINTS[SupportedChainId.MAINNET]?.[0],
-    [props.jsonRpcEndpoint]
-  )
+  const defaultChainId = useMemo(() => {
+    if (!props.defaultChainId) return 1
```

Comment:
would make this a var outside the component - `const DEFAULT_CHAIN_ID = SupportedChainId.Mainnet`